app/controllers/calagator/application_controller.rb

```diff
@@ -28,6 +29,10 @@ def recaptcha_enabled?
 
     protected
 
+    def spam_detector_enabled?
```

Comment:
Where is this being used? Maybe I'm just missing it.

Reply:
You're right - I didn't fully finish this thought - standby